HTML/HTML.sublime-syntax

```diff
@@ -260,7 +260,7 @@ contexts:
         - include: style-css-content
 
   style-css-content:
-    - match: ''
```

Comment:
The current implementation of HTML doesn't support wrapping CSS in html comments (`<!-- ... -->`) That's why the match is empty. It is by intent!

Those changes may fix some bugs, but are not within scope of this PR.

If we want to change the behavior it should go into a separate one as it results in further changes.

I'd like to come up with an implementation, which handles script/style tags the same way, but it should also fix some issues related with those html comments.

Reply:
Can you provide an example that this change breaks?

Reply:
```
        <style> h1 {} </style>
        ##     ^^^^^^^ source.css.embedded.html
```

... fails as `source` starts with `h1`. The space before is unscoped.

Reply:
Moving to the other discussion below, since it's about the same topic.